### DISCUSSION TITLE
Correct way to use animations with useAnimations

### DISCUSSION THREAD

**Discussion by Rguarnizo** [pmndrs/drei#435] [Q&A]
### What is your question:

Hello, community and developers, I would like to know what is the right way to animate models with UseAnimatios hook, I'm trying to encourage the following [SketchFab model](https://sketchfab.com/3d-models/baba-yagas-hut-479b5f3c493349b18c59dbcbe354d98f), I export it in GLTF format and I uploaded it to [Codesandbox ](https://codesandbox.io/s/intelligent-blackburn-ng2gz?file=/src/BabaYaga.js)(wait to loading),I try to animate it but it does it in a strange way. 

What is the correct form to Animate this models? I try to animate a models with single NLA Tracks (Blender) and UseAnimations works correctly. But when i have more NLA Tracks not works fine...

Thanks everyone for your help.

**Comment by joshuaellis** [comment]
This feels like more of a question on how to use blender...? But if you have a model and codesandbox, someone might be able to take a look
